App/Redux/LoginActions.ts

```diff
@@ -0,0 +1,70 @@
+/// <reference path="../../typings/tsd.d.ts" />
+
+export interface ISettings {
```

Comment:
Lets discuss on namings after the scrum. We need to be consistent and more explicit in naming in order to understand the purpose of the interfaces and functions.